### PR TITLE
add text around species assignment

### PR DESCRIPTION
**Pull request by hardingnj**
Add text around species assignment

### CODE REVIEW

File: content/02.results.md [hunk 1]
```diff
@@ -62,6 +60,48 @@ Full details including exclusion thresholds are available in supplementary.
 
 Summary of site coverage post QC exclusions.
 
+### Species assignment and sex calling
+
+The Anopheles gambiae complex is a crypic group of sibling species, with no single locus offering unambiguous resolution of species.
```
Comment (cclarkson) [inline content/02.results.md:65]: Going to need some good refs here - probably something like (Scott, Fanello, Santolamazza, but see Vincente, 'some cryptic species papers', Ag1000G Consortium papers). e.g. Yes, there are lots of markers that folk claim are good, but when we use them to look widely across Africa, they're not always helpful.

File: notebooks/README [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+- sampleqc: Quality control of samples
+- sequencing: Technical details of sequnce
+- site-filters: Accessibility and genome QC
+- snpdiscovery: Related to counts of SNPs, segregating sites
+- popstructure: Analyses of population structure ie PCA etc
+
+Then an undercore; second and subsequent words express more detail about notebook separated by hyphen.
+
+`{results-topic}_{more-detail-here}.ipynb`
+
```
Comment (cclarkson) [inline notebooks/README:16]: noted.

**Comment by cclarkson** [comment]
@hardingnj  - other than ref comment, this is looking good. Can be merged if you like and we can come back to refs?

**Comment by hardingnj** [comment]
Thanks Chris- let's merge and come back to the species references.

**Pull request merged**